### PR TITLE
Remove add fragments from change set when they are not referenced from any other fragment

### PR DESCRIPTION
**Pull request by artman**

### CODE REVIEW

**Comment by artman** [comment]
Hmm, failing?

File: Jetstream/ChangeSet.swift [hunk 1]
```diff
@@ -79,22 +79,48 @@ public enum ChangeSetState {
         self.procedure = procedure
         self.atomic = atomic
         
+        var attachedObjectUUIDs = [String]();
+        var addedObjects = [String: SyncFragment]();
```
Comment (robskillington) [inline Jetstream/ChangeSet.swift]: Nit: semi-colons

**Comment by robskillington** [comment]
Other than semi-colons LGTM after test fixes =]

File: Jetstream/ChangeSet.swift (hunk 1 above)
Comment (artman) [inline Jetstream/ChangeSet.swift]: Haha, yeah, have been writing ObjC for a while again.

**Pull request merged**